frontend/src/__tests__/cypress/cypress/e2e/userManagement/userManagement.cy.ts

```diff
@@ -0,0 +1,62 @@
+import { mockGroupSettings } from '~/__mocks__/mockGroupConfig';
+import { userManagement } from '~/__tests__/cypress/cypress/pages/userManagement';
+import { asProductAdminUser, asProjectAdminUser } from '~/__tests__/cypress/cypress/utils/users';
+import { pageNotfound } from '~/__tests__/cypress/cypress/pages/pageNotFound';
+
+it('Cluster settings should not be available for non product admins', () => {
+  asProjectAdminUser();
+  userManagement.visit(false);
+  pageNotfound.findPage().should('exist');
+  userManagement.findNavItem().should('not.exist');
+});
+
+describe('User Management', () => {
+  beforeEach(() => {
+    asProductAdminUser();
+    cy.interceptOdh('GET /api/groups-config', mockGroupSettings());
+    cy.interceptOdh('PUT /api/groups-config', mockGroupSettings()).as('saveGroupSetting');
+    userManagement.visit();
+  });
+
+  it('Administrator group setting', () => {
+    const administratorGroupSection = userManagement.getAdministratorGroupSection();
+    administratorGroupSection.shouldHaveAdministratorGroupInfo();
+    administratorGroupSection.clearMultiChipItem();
+    administratorGroupSection.selectMultiGroup('odh-admins');
+    administratorGroupSection.findMultiSelectGroupInput().type('odh-admin');
+    administratorGroupSection.findMultiGroupOptions('odh-admins-1').click();
+    administratorGroupSection.findChipItem('odh-admins').should('exist');
+    administratorGroupSection.removeChipItem('odh-admins');
+    administratorGroupSection.findChipItem(/^odh-admins$/).should('not.exist');
+    administratorGroupSection.removeChipItem('odh-admins-1');
+    administratorGroupSection.findErrorText().should('exist');
+    administratorGroupSection.findMultiGroupOptions('odh-admins').click();
+    administratorGroupSection.findErrorText().should('not.exist');
+    userManagement.findSubmitButton().should('be.disabled');
+  });
+
+  it('User group setting', () => {
+    const userGroupSection = userManagement.getUserGroupSection();
+    userManagement.findSubmitButton().should('be.disabled');
+    userGroupSection.findErrorText().should('exist');
```

Comment:
This is an invalid state -- which is fine to test, but you should test success states that the App goes through.